consensus/src/round_manager.rs

```diff
@@ -126,7 +126,7 @@ impl<T: Payload> RecoveryManager<T> {
             network,
             storage,
             state_computer,
-            ledger_recovery_data,
+            last_committed_round: ledger_recovery_data.commit_round(),
```

Comment:
Given we only extract the last_committed_round, does it make sense to just pass the round in rather than the full LedgerRecoveryData?